packages/contracts/contracts/L2/ValidatorRewardVault.sol

```diff
@@ -144,6 +144,28 @@ contract ValidatorRewardVault is FeeVault, ISemver {
         require(success, "ValidatorRewardVault: ETH transfer failed");
     }
 
+    /**
+     * @notice Withdraws all remaining contract balance excluding reserved amount to ProtocolVault.
+     *         Reverts if the withdraw amount is less than the minimum withdrawal amount.
+     */
+    function withdrawToProtocolVault() external {
+        uint256 amount = address(this).balance - totalReserved;
+        require(
+            amount >= MIN_WITHDRAWAL_AMOUNT,
+            "ValidatorRewardVault: withdrawal amount must be greater than minimum withdrawal amount"
+        );
+        require(amount > 0, "ValidatorRewardVault: withdrawal amount must be greater than zero");
```

Comment:
Seems MIN_WITHDRAWAL_AMOUNT is zero at mainnet, so it seems like dup. Can we just leave the second require statement?